<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CfgSettingNetworkStreamModeSelect</class>
 <widget class="QWidget" name="CfgSettingNetworkStreamModeSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>284</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <widget class="QPushButton" name="pushButton_cancel">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Output</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Network 1-8</string>
      </property>
     </widget>
     <widget class="QComboBox" name="output1_8">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>40</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>AES67</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dante</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Network 9-16</string>
      </property>
     </widget>
     <widget class="QComboBox" name="output9_16">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>70</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>AES67</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dante</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_ok">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>100</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Input</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="input1_8">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>40</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>AES67</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dante</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="input9_16">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>70</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>AES67</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dante</string>
       </property>
      </item>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
